src/file_import.py

```diff
@@ -48,8 +46,12 @@ def _import_from_file(self, file, import_settings):
             callback = file_io.import_from_xry
         case _:
             callback = file_io.import_from_columns
-    item = callback(self, file, import_settings)
-    item.name = import_settings.name
-    if not item.name:
-        item.name = filename
-    return item
+    items = callback(self, file, import_settings)
+    if not import_settings.name:
+        i = 0
+        for item in items:
+            if i < 1:
+                item.name = filename
+            else:
+                item.name = f"{filename} ({i})"
```

Comment:
This will never reach the else condtionaly, as i is never changed upon iterations. i will always be zero.
Also, this logic is already handled in the add_data function.

Reply:
You're absolutely right. My initial thinking was, if a file would extract multiple items, that they are always added, regardless of what the user configured while handling duplicates. A better solution would be to name the individual items something like "*filename* - Voltage" and "*filename* - Amperage" and handle that inside the respective function.

Removing this is probably for the better